lib/ledger_sync/quickbooks_online/deserializer.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+# Gem.find_files('ledger_sync/ledgers/quickbooks_online/serializer_type/**/*.rb').each { |path| require path }
+
+module LedgerSync
+  module QuickBooksOnline
+    class Deserializer < LedgerSync::Deserializer
+      def deserialize(args = {})
+        hash                  = args.fetch(:hash)
+        merge_for_full_update = args.fetch(:merge_for_full_update, false)
+        resource              = args.fetch(:resource)
+
+        deserialized_resource = super(hash: hash, resource: resource)
+
+        # Ref: https://github.com/LedgerSync/ledger_sync/issues/86
+        if deserialized_resource.is_a?(LedgerSync::QuickBooksOnline::Account) &&
+           deserialized_resource.AccountType &&
+           deserialized_resource.Classification.blank?
+          deserialized_resource.Classification = deserialized_resource.class::TYPES_TO_CLASSIFICATION_MAPPING.fetch(
+            deserialized_resource.AccountType,
+            nil
+          )
+        end
+
+        return deserialized_resource unless merge_for_full_update
+
+        merge_resources_for_full_update(
+          hash: hash,
+          ledger_resource: deserialized_resource,
+          resource: resource
+        )
+      end
+
+      def merge_resources_for_full_update(args = {})
+        hash            = args.fetch(:hash)
+        ledger_resource = args.fetch(:ledger_resource)
```

Comment:
`ledger_resource` is redeclared in the next few lines, so this isn't used.

Reply:
Nice catch!

Reply:
@jozefvaclavik can you resolve this and then let's get this merged?

Reply:
@amouxkin @ryanwjackson Done.